Feature Campañas

larochem20 opened the pull request:
Creacion de la carpeta campaña y el archivo campaña.js en la carpeta models. Creacion de la carpeta campaña y el archivo campaña.js en la carpeta controllers Creacion de la carpeta campaña y el archivo campaña.js en la carpeta routes

larochem20 committed:
Creacion carpetas/ archivos js

Creacion de la carpeta campaña y el archivo campaña.js en la carpeta models.
Creacion de la carpeta campaña y el archivo campaña.js en la carpeta controllers
Creacion de la carpeta campaña y el archivo campaña.js en la carpeta routes

LuisJuarez17 committed:
Creación controladores campañas

Creación de controladores para campañas

Endercarias committed:
Creación de las rutas de campañas

Creación de las rutas de campañas

HJOC15 reviewed (approved):
Revisión completada de los controladores y rutas de campañas y no hay ningún error.

Por favor ser más específico a la hora de colocarle nombre al Pull Request.

DJPORTILLO30 commented:
Editar destino de pull request ya que va hacia la rama master , necesita ir hacia la rama test-branch! @LuisJuarez17 @larochem20 @Endercarias .

SFMZ21 reviewed (changes requested):
Asignar nombres descriptivos a los controladores y rutas, utilizando Campaign en lugar de item, por ejemplo: getCampaign, createCampaign.

HJOC15 reviewed (changes requested):
Por favor hacer el cambio de la rama Master a la de Test.

Endercarias committed:
Creación de modelo

Creación del modelo campañas, modificación de controladores y documentación de rutas.

Co-Authored-By: Luis Juárez <49957257+LuisJuarez17@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

HJOC15 commented on models/campañas/campañas.js:
```diff
@@ -0,0 +1,38 @@
+Campaign.find = Campaign.findAll;
+Campaign.findById = Campaign.findByPk;
+
+module.exports = Contact;
```
Hicieron mal el copypaste y están exportando contactos, por favor revisar y corregir.

HJOC15 reviewed (changes requested):
Revisar la línea de código comentada.

SFMZ21 reviewed (changes requested):
Revisar el modelo, ya que se está exportando "Contact"?

DJPORTILLO30 reviewed (changes requested):
Modifcar el modelo de campaña

larochem20 committed:
Corrección del archivo exportado 

Se modificó en module.exports = Contact 
A module.exports = Campaign

HJOC15 reviewed (approved):
Aprobado!

SFMZ21 reviewed (approved):
Todo ok

DJPORTILLO30 reviewed (approved):
Todo okay
